log_subscriber/internal/infrastructure/redis/redis.go

```
@@ -0,0 +1,48 @@
+	if err != nil {
+		return fmt.Errorf("failed to marshal event: %w", err)
+	}
+
+	// Set TTL as 60 minutes
+	if err := r.client.Set(ctx, key, serialized, 60*time.Minute).Err(); err != nil {
+		return err
+	}
+	return nil
+}
+
+func valOrEmpty(strPtr *string) string {
```

golangci-lint (1.62.2)

func `valOrEmpty` is unused

(unused)